src/main/resources/static/resources/subscription_templates.js

```diff
@@ -16,6 +16,12 @@ var template_vars = {};
                         "formvalue" : ""
                     }
                 ],
+            "notificationMessageKeyValuesAuth" : [
```

Comment:
I thought all subscription templates was moved to EI Backend since some months back. If you use  and update this template you will have to update EI Backend with same changes. There is RestApi to retrieve the Subscription template from EI Backend. I suggest to change so Subscription template content is read from EI Backend, so we use one and same subscription template in all places and we will then have only one subscription template file to keep updated.